.github/workflows/ci.yml

```diff
@@ -38,47 +38,111 @@ jobs:
+    - name: Update version in rdmo/__init__.py
+      run: |
+        sed -i "s/__version__ = .*/__version__ = \"${{ steps.new-version.outputs.new_version }}\"/" rdmo/__init__.py
+    # build the webpack bundle
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 18
+        cache: npm
+    - run: npm install && npm run build:prod
+    # build the wheel
+    - uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+        cache: pip
+    - run: |
+        python -m pip install --upgrade pip build[uv] twine
+        python -m pip --version
+    - name: Build the wheel
+      run: python -m build --installer=uv
+    - name: Check the metadata of wheel and sdist
+      run: python -m twine check --strict dist/*
+    - name: Install package from built wheel
+      run: python -m pip install --no-compile dist/rdmo*.whl # do not create __pycache__/*.pyc files
+    - name: Write info to step summary
+      run: |
+          {
+            echo -e "# ✓ Wheel successfully built (v${{ steps.new-version.outputs.new_version }})\n\n"
+            echo '<details><summary>Information about installed wheel</summary>'
+            echo -e "\n\`\`\`console"
+            echo "$ python -m pip show --files --verbose rdmo"
+            python -m pip show --files --verbose rdmo
+            echo -e "\`\`\`\n</details>"
+          } >> $GITHUB_STEP_SUMMARY
+    - name: Upload wheel as artifact
+      uses: actions/upload-artifact@v4
+      with:
+        name: wheel
+        path: dist/rdmo*.whl
+        if-no-files-found: error
+        retention-days: 30
+
   test:
+    name: "Test (Python: ${{ matrix.python-version }}, DB: ${{ matrix.db-backend }})"
+    needs: build-wheel
     runs-on: ubuntu-24.04
     strategy:
       matrix:
         python-version: ['3.8', '3.12']
         db-backend: [mysql, postgres]
-    name: "Test (Python: ${{ matrix.python-version }}, DB: ${{ matrix.db-backend }})"
-    needs: lint
     steps:
     - uses: actions/checkout@v4
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v5
       with:
         python-version: ${{ matrix.python-version }}
         cache: pip
+    - name: Download wheel
+      uses: actions/download-artifact@v4
+      with:
+        name: wheel
+        path: dist
     - name: Install Dependencies
       run: |
-        sudo apt update
-        sudo apt install --yes pandoc texlive-xetex librsvg2-bin
-        python -m pip install --upgrade pip
+        sudo apt-get update && sudo apt-get install --yes pandoc texlive-xetex librsvg2-bin
         pandoc --version
-    - name: Install rdmo[mysql] and start mysql
+        python -m pip install --upgrade pip
+        python -m pip --version
+    - name: Install rdmo[mysql] from wheel and start mysql
       run: |
-        python -m pip install --editable .[ci,mysql]
+        python -m pip install "$(ls dist/*.whl)[ci,mysql]"
```

Comment:
Now we actually the wheel instead of the editable install from the repository.

This is more like a "production" installation, like `pip install rdmo`, where users don't editable install from source.

Reply:
Are you sure that does not mess with the coverage? Or is this a different job. (I had this case once...)